umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso_shared/distributions_related_url.clj

```diff
@@ -138,47 +143,48 @@
                      (str/trim (subs mime-type (inc (.indexOf mime-type ":"))))))})))
 
 (defn parse-online-urls
-  "Parse ISO online resource urls"
+  "Parse ISO online resource urls. There are several places in ISO that allow for multiples of'
+   online urls."
   [doc sanitize? service-urls distributor-xpaths-map]
-  (for [distributor (select doc (get distributor-xpaths-map :Root))
-        :let [size (value-of distributor (str (get distributor-xpaths-map :TransferOptions)
-                                              "/gmd:transferSize"))
-              unit (value-of distributor (str (get distributor-xpaths-map :TransferOptions)
-                                              "/gmd:unitsOfDistribution/gco:CharacterString"))
-              fees (value-of distributor (get distributor-xpaths-map :Fees))
-              url (get distributor-xpaths-map :URL)
-              format (parse-distributor-format distributor distributor-xpaths-map)
-              name (char-string-value distributor (str url "gmd:name"))
-              code (value-of distributor (str url "gmd:function/gmd:CI_OnlineFunctionCode"))
-              url-link (value-of distributor (str url "gmd:linkage/gmd:URL"))
-              url-link (when url-link (url/format-url url-link sanitize?))
-              opendap-type (when (= code "GET DATA : OPENDAP DATA (DODS)")
-                            "USE SERVICE API")
-              types-and-desc (parse-url-types-from-description
-                              (char-string-value distributor (str url "gmd:description")))
-              service-url (first (filter #(= url-link (:URL %)) service-urls))
-              type (or opendap-type (:Type types-and-desc) (:Type service-url))]
-        :when (seq type)]
-    (merge
-     {:URL url-link
-      :URLContentType "DistributionURL"
-      :Type type
-      :Subtype (if opendap-type
-                "OPENDAP DATA (DODS)"
-                (or (:Subtype types-and-desc) (:Subtype service-url)))
-      :Description (:Description types-and-desc)}
-     (case type
-       "GET DATA" {:GetData {:Format (su/with-default (:Format format) sanitize?)
-                             :Size (if sanitize?
-                                     (or size 0.0)
-                                     size)
-                             :Unit (if sanitize?
-                                     (or unit "KB")
-                                     unit)
-                             :Fees fees
-                             :Checksum (:Checksum types-and-desc)
-                             :MimeType (:MimeType format)}}
-       nil))))
+  (flatten
+    (for [distributor (select doc (get distributor-xpaths-map :Root))
+          :let [fees (util/trunc (value-of distributor (get distributor-xpaths-map :Fees)) size-of-related-url-fees)
+                format (parse-distributor-format distributor distributor-xpaths-map)]]
+      (for [transferOption (select distributor (get distributor-xpaths-map :TransferOptions))
```

Comment:
I think in this case it would be good to use normal Clojure conventions (dash-case) and call it `transfer-option`.

Reply:
fixed